docs/tutorials/heatmaps.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Can you color a subset of the edges? Or does it have to be all? The first three examples are straightforward enough but this could maybe benefit from some short explanatory text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
added an explanatory text.